### ISSUE TITLE
iPhone 5s ios 12.4.5

### ISSUE THREAD

**Issue by maxii26** [checkra1n/BugTracker#1173]
**Tell us about your setup:**
1. What iDevice are you using?
iPhone 5s Global
2. On what version of iOS is it?
iOS 12..4.5
3. What version of checkra1n are you using?
0.10.1
4. What is your host system (OS version? Hackintosh? VM? etc.)?
BootRa1n in VM and USB
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
USB-A to Lightning 3rd Party cable
**What are the steps to reproduce the issue?**
1. Boot on Bootra1n
2. Follow the steps to do jailbreak
3.stuck on right before trigger
...

**What do you expect, and what is happening instead?**
I expect do jailbreak but it still stuck in there

**Does the issue also occur if you enable Safe Mode?**
yes

**Any other info, error logs, screenshots, ...?**
![image](https://user-images.githubusercontent.com/50774597/78901651-a6070a00-7a46-11ea-9366-7e5411341d79.png)

help pls i have like 4 months without jailbreak :c

**Comment by CanerKaraca23** [comment]
Unplug and plug fastly your device and wait then jailbreak successful.

**Comment by maxii26** [comment]
> Unplug and plug fastly your device and wait then jailbreak successful.

in what moment i do that?
in the "right before trigger"?

**Comment by maxii26** [comment]
> Unplug and plug fastly your device and wait then jailbreak successful.

it worked but the checkra1n doesn't appear in the homescreen

**Comment by maxii26** [comment]
Sorry the quality but 
![image](https://user-images.githubusercontent.com/50774597/78904341-5aeef600-7a4a-11ea-8e28-7f7be2a981ca.png)

**Comment by CanerKaraca23** [comment]
Yes same issue.

**Comment by RoanneGrech** [comment]
similar issue except the screen on my ipad is remaining black. when i click done on ckeckrain, it says that it's in dfu mode

- this worked for me
https://www.reddit.com/r/jailbreak/comments/e86xmc/tutorialtip_checkra1n_jailbreak_beta_096_error_31/

basically, I found the package contents, went in MacOS folder, there is a file called checkra1n. I dragged and dropped it in Terminal. started the program from there, set the verbose boot and voila, it takes a couple of seconds to appear after booting the device but it worked like a charm
